### PR TITLE
fix: update the pmlog_summary to support new metrics

### PR DESCRIPTION
**Pull request by xiangce**
Signed-off-by: Xiangce Liu <xiangceliu@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [x] Is this PR an enhancement?

### Complete Description of Additions/Changes:

Enhance the PmLogSummary for the following lines of new metrics.
```
        kernel.all.pressure.io.full.avg ["10 second"] 0.001 none
        kernel.all.pressure.cpu.some.avg ["1 minute"] 14.942 none
        kernel.all.pressure.memory.full.avg ["5 minute"] 0.002 none
        disk.dev.total ["vda"] 0.016 count / sec
        disk.dev.total ["vdb"] 0.445 count / sec
        disk.dev.total ["vdc"] 2.339 count / sec
```

### CODE REVIEW

File: insights/parsers/pmlog_summary.py [hunk 1]
```diff
@@ -70,24 +75,43 @@ class PmLogSummary(CommandParser, dict):
         kernel.all.cpu.nice  0.000 none
         kernel.all.cpu.steal  0.000 none
         kernel.all.cpu.idle  3.986 none
+        kernel.all.pressure.io.full.avg ["10 second"] 0.001 none
```
Comment (apuntamb) [inline insights/parsers/pmlog_summary.py:78]: @xiangce based on these [metrics](https://gitlab.cee.redhat.com/pgarciaq/rosrules/-/blob/master/samples/aws2json.py#L90-L96), do we need to keep the older ones as is?

Comment (xiangce) [reply insights/parsers/pmlog_summary.py:78]: @apuntamb I think it's OK, since it's just an example here.

**Review by kgaikwad — APPROVED** [review]
Thank you @xiangce!
Though I haven't  tested, code wise changes look good to me :+1: 

@apuntamb & @bfahr - do you have any additional comments?

**Comment by apuntamb** [comment]
@xiangce Tested locally, works as expected! Thanks for helping with the testing.
@kgaikwad Looks good to me! :100:

**Pull request merged**